docs/developers/tensorflow_coverage/tflite_base_converage.md

```diff
@@ -0,0 +1,165 @@
+# TensorFlow Lite Base Operation Coverage
```

Comment:
How do you feel about also deleting the old coverage tables in this https://github.com/google/iree/tree/main/docs/developers/tensorflow_coverage folder? Those were all last updated 2020/12/8...

Reply:
I honestly dont know enough about those charts to know if they are still useful. They are nice in that they are generated by a script and more map tensorflow support whether directly in IREE or via TFLite.

Reply:
Well, they _were_ generated by a script. The code/tests used to generate them is about 3 iterations of testing infrastructure old :P

I think we can just delete them. If anyone still finds them useful, we/they should bring them back using the latest code.